pkg/operator/starter.go

```diff
@@ -235,6 +234,13 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		kubeClient.CoreV1(),
 		kubeClient.CoreV1(),
 		kubeInformersForNamespaces,
+	).WithAuditPolicyController(
+		operatorclient.TargetNamespace,
+		"audit",
```

Comment:
ok, this configmap will be mounted here https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/bindata/v3.11.0/openshift-apiserver/deploy.yaml#L167